compliance-web/src/components/App/Inspections/Profile/Enforcements/ViolationTicket/ViolationTicketUpdateModal.tsx

```diff
@@ -158,7 +158,7 @@ const ViolationTicketUpdateModal: FC<ViolationTicketUpdateModalProps> = ({
               label="Ticket #"
               placeholder="Enter ticket number"
               sx={{ width: "100%" }}
-
+              disabled
```

Comment:
Does this work in case if we have a valid requirement. Should this be conditional?